static/scss/resource-embed-widget.scss

```diff
@@ -0,0 +1,14 @@
+.content-embed-widget {
```

Comment:
Some of the naming here feels a bit confusing.  This component is referred to both as a "site content embed widget" and a "resource embed widget."  "Resource embed widget" feels more correct but it also refers to a type of content defined in a config file that could change.  Either way, I think we should refer to it as one name to reduce confusion.

Reply:
oh yeah I think I sort of changed my mind and forgot to go through fully. I'll just call it resource embed widget

Reply:
I went through and renamed everything in line with that